doc/classes/AStarGrid2D.xml

```diff
@@ -12,105 +23,156 @@
+			The offset of the grid which will be applied to calculate the resulted point position returned by the [method get_point_path]. After changed, the [method update] needs to be called before using the [AStarGrid2D] to calculate path.
 		</member>
 		<member name="size" type="Vector2i" setter="set_size" getter="get_size" default="Vector2i(0, 0)">
+			The size of the grid. After changed, the [method update] needs to be called before using the [AStarGrid2D] to calculate path.
 		</member>
 	</members>
 	<constants>
 		<constant name="HEURISTIC_EUCLIDEAN" value="0" enum="Heuristic">
+			The Euclidean heuristic to be used for the pathfinding using the following formula:
+			[codeblock]
+			dx = abs(to_id.x - from_id.x)
+			dy = abs(to_id.y - from_id.y)
+			result = sqrt(dx * dx + dy * dy)
+			[/codeblock]
 		</constant>
 		<constant name="HEURISTIC_MANHATTAN" value="1" enum="Heuristic">
+			The Manhattan heuristic to be used for the pathfinding using the following formula:
+			[codeblock]
+			dx = abs(to_id.x - from_id.x)
+			dy = abs(to_id.y - from_id.y)
+			result = dx + dy
+			[/codeblock]
 		</constant>
 		<constant name="HEURISTIC_OCTILE" value="2" enum="Heuristic">
+			The Octile heuristic to be used for the pathfinding using the following formula:
+			[codeblock]
+			dx = abs(to_id.x - from_id.x)
+			dy = abs(to_id.y - from_id.y)
+			f = sqrt(2) - 1
+			result = (dx &lt; dy) ? f * dx + dy : f * dy + dx;
+			[/codeblock]
 		</constant>
 		<constant name="HEURISTIC_CHEBYSHEV" value="3" enum="Heuristic">
+			The Chebyshev heuristic to be used for the pathfinding using the following formula:
+			[codeblock]
+			dx = abs(to_id.x - from_id.x)
+			dy = abs(to_id.y - from_id.y)
+			result = max(dx, dy)
+			[/codeblock]
 		</constant>
 		<constant name="HEURISTIC_MAX" value="4" enum="Heuristic">
+			Represents the size of the [enum Heuristic] enum.
 		</constant>
 		<constant name="DIAGONAL_MODE_ALWAYS" value="0" enum="DiagonalMode">
+			The searching algorithm will ignore solid neighbors around the target cell and allow passing using diagonals.
 		</constant>
 		<constant name="DIAGONAL_MODE_NEVER" value="1" enum="DiagonalMode">
+			The searching algorithm will completely ignore diagonals and the way will be always orthogonal.
 		</constant>
 		<constant name="DIAGONAL_MODE_AT_LEAST_ONE_WALKABLE" value="2" enum="DiagonalMode">
+			The searching algorithm will avoid using diagonals if at least two obstacles has been placed around the neighboring cells of the specific path segment.
 		</constant>
 		<constant name="DIAGONAL_MODE_ONLY_IF_NO_OBSTACLES" value="3" enum="DiagonalMode">
+			The searching algorithm will avoid using diagonals if any obstacle has been placed around the neighboring cells of the specific path segment.
 		</constant>
 		<constant name="DIAGONAL_MODE_MAX" value="4" enum="DiagonalMode">
+			Represents the size of the [enum DiagonalMode] enum.
```

Comment:
```suggestion
			The size of the [enum DiagonalMode] enum.
```

Reply:
I did it cuz other enums across the engine uses such words fox example: `Represents the size of the [enum TextureParam] enum.` in the BaseMaterial3D.xml.

Reply:
Ah, keep these then :)